app/models/outcome.rb

```diff
@@ -23,4 +23,8 @@ class Outcome < ActiveRecord::Base
   def to_s
     "#{name} - #{description}"
   end
+
+  def label
+    "#{name} - #{nickname}"
```

Comment:
This is a good idea.

If we want "Name" to be upper case everywhere, we're probably better off handling that in a way that makes sure it's upper case going into the database. There's a few different options we can use for that. I can follow up with another PR for it.